ios/Client/EXHomeAppManager.m

```diff
@@ -126,12 +126,7 @@ - (NSString *)bundleResourceNameForAppFetcher:(__unused EXAppFetcher *)appFetche
 
 - (BOOL)appFetcherShouldInvalidateBundleCache:(__unused EXAppFetcher *)appFetcher
```

Comment:
I would remove this entirely for Home. (a) we might not use remote updates for Home and (b) expo-updates has some checks to make sure we can run the JS before it decides to commit to using the latest update.